android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeFragment.kt

```diff
@@ -82,12 +105,31 @@ class PracticeFragment : Fragment() {
             viewModel.onCorrectComplete()
         })
 
+        hintButton.setOnClickListener {
+            viewModel.onHint()
+            Toast.makeText(context, viewModel.getDotsString(), Toast.LENGTH_SHORT).show()
+            Timber.i("Hint invoked")
+            val expectedDots = viewModel.getExpectedDots()

```

Comment:
It is not needed here

Reply:
could you please be more specific? Each line of code has its meaning and removal will make our app act differently, so what is 'it' and where is it needed if not here?